pest_derive/src/optimizer.rs

```diff
@@ -137,14 +137,12 @@ pub fn optimize(rules: Vec<Rule>) -> Vec<Rule> {
 mod tests {
     use super::*;
 
-    use quote::Ident;
-
     #[test]
     fn concat_strings() {
         let rules = vec![
             Rule {
-                name: Ident::new("rule"),
-                ty: RuleType::Atomic,
+                name: "rule",
+                ty:   RuleType::Atomic,
```

Comment:
Space here is not needed per rustfmt.

Reply:
If I run rustfmt on this file, I get a bunch of changes (many not related to the changes I made) Should I just go ahead and use rustfmt, or should I just remove these spaces? I am using rustfmt 0.8.0